modules/server/src/main/scala/seqexec/server/tcs/TcsSouthControllerEpicsAo.scala

```diff
@@ -213,27 +207,34 @@ object TcsSouthControllerEpicsAo {
       current:    EpicsTcsAoConfig,
       demand:     GemsGuiders
     ): List[WithDebug[EpicsTcsAoConfig => F[EpicsTcsAoConfig]]] = List(
-      current.mapping
-        .get(Cwfs1)
-        .flatMap(setCwfs1Guide(_, subsystems, current.cwfs1.tracking, demand.cwfs1.tracking)),
-      current.mapping
-        .get(Cwfs2)
-        .flatMap(setCwfs2Guide(_, subsystems, current.cwfs2.tracking, demand.cwfs2.tracking)),
-      current.mapping
-        .get(Cwfs3)
-        .flatMap(setCwfs3Guide(_, subsystems, current.cwfs3.tracking, demand.cwfs3.tracking)),
-      current.mapping
-        .get(Odgw1)
-        .flatMap(setOdgw1Probe(_)(subsystems, current.odgw1.tracking, demand.odgw1.tracking)),
```

Comment:
The parking was implemented for ODGW, but this logic implied that the ODWG would be configured only if it was included in the Virtual Telescope Mapping, which never happened if the telescope configuration required it to be parked!